lib/Raven/Serializer.php

```diff
@@ -67,17 +67,44 @@ public function __construct($mb_detect_order = null)
      * sanitization and encoding.
      *
      * @param mixed $value
+     * @param array $context
+     * @return string|bool|double|int|null|object|array
```

Comment:
`mixed` can be used when a value can be literally anything; the author of the documentation is unable to predict which type it will be. So, instead of having a long list of types you can just use that type. There is no reason to not do it